content/post/28-sept-21-oss.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Welcome to OSDC Newsletter!"
+---
+
+<!-- Introduction to Newsletter -->
+
+We will be introducing y'all with the world of free and open source software this week.
+
+# Open Source Explainers
```

Comment:
Suggest content related to Open Source Explanations

Reply:
https://opensource.com/resources/what-open-source
https://kushaldas.in/pages/hacker-ethic-and-free-software-movement.html

Reply:
What is Open Source explained in LEGO - https://www.youtube.com/watch?v=a8fHgx9mE5U